configure.ac

```diff
@@ -570,6 +570,13 @@
         AC_DEFINE([HAVE_DETECT_DISABLED], [1], [Detection is disabled])
     ])
 
+  # enable response time calculations
```

Comment:
I would like to avoid a configure time option for what can easily be a runtime option. Configure time options are really only meant for features that add new dependencies.

Reply:
OK, I made it a configure time option because I thought adding a runtime option would incur some overhead to check if the feature was enabled or not during the run of the flow engine. I can make it a runtime option instead though.

Reply:
Fixed with recent change